db/clientlist_virtual_reader.hh

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2019 ScyllaDB
+ *
+ * Modified by ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "flat_mutation_reader.hh"
+#include "mutation_reader.hh"
+#include "schema.hh"
+#include <seastar/core/distributed.hh>
+#include <seastar/core/shared_ptr.hh>
+
+namespace cql_transport {
+class cql_server;
+}  // namespace cql_transport
+
+namespace db::system_keyspace {
+    struct client_data;
+}  // namespace system_keyspace
+
+namespace db::clientlist {
+
+class clientlist_mutation_reader final : public flat_mutation_reader::impl {
```

Comment:
Please write a comment explaining what data this reader generates.
Here or above the virtual_reader struct, wherever you think would be more suitable.